### ISSUE TITLE
Use Big-endian not Little-endian to convert 32-bit integers to 4 bytes, and back

### ISSUE THREAD

**Issue by aisrael** [pyrige/skippy#1]
```
    def encrypt(self, value):
        buf = struct.pack("=I", value)
        enc = skip32(self._key, buf, True)
        return struct.unpack("=I", enc)[0]
```

Cause this Python Skip32 code to produce inconsistent results with the Java implementation at https://github.com/boivie/skip32-java/blob/55f3e5bcd6c6facf2f44d472df5cf80a537e5697/src/main/java/com/boivie/skip32/Skip32.java

On the other hand, using 

```
    def encrypt(self, value):
        buf = struct.pack(">I", value)
        enc = skip32(self._key, buf, True)
        return struct.unpack(">I", enc)[0]
```

Makes it consistent.

**Comment by pyrige** [comment]
I'd rather not change that from native byte order (`=I`) to anything specific. It's not exactly ideal that skip32 yields different results based on endianess but it's what the original C implementation does.

Maybe I'm missing something here, but you can also check against https://github.com/gray/crypt-skip32-xs/blob/master/skip32.c on different platforms. Either way, I added unit tests to make it clear that parity with skip32 is the intention.

**Comment by aisrael** [comment]
Ahh... I understand. I was just surprised that even when I ported the Java implementation to Dart, both Dart and Java were consistent but this Python implementation was the only outlier. Finally managed to trace it to endian-ness.

No worries, I just ended up using a 'local fork' of `skippy.py` in our project and used `>L` so it would agree with all the other implementations for all the other platforms.
